staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -66,6 +78,13 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 		// All authorization builder and accessor functions have a nominal cost
 		cost := uint64(1)
 		return &cost
+	case "fieldSelector", "labelSelector":
+		// field and label selector parse is a string parse into a structured set of requirements
+		if len(args) >= 2 {
```

Comment:
@jpbetz it's a sharp edge that in CallCost the function args are arg[0]==receiver, arg[1+] are the args, and in EstimateCallCost, target is the receiver and args are the args ... any way we can make that more visible or clearer?

Reply:
We (CEL devs) can follow up on this one. I've opened https://github.com/kubernetes/kubernetes/issues/125775 to track.